developer-docs-site/docs/index.md

```diff
@@ -55,7 +57,7 @@ Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentatio
 </div>
 </div>
 
-## Nodes
+## Run Aptos validator and fullnodes
```

Comment:
This is a bit tricky and I struggled a lot during initial days here. There are these:
- validator node
- validator fullnode
- public fullnode
So a quick read of the above change can make the reader think "so are these fullnodes public fullnodes or validator fullnodes?" for a second or two, which means we have already conveyed confusion. So simply saying "Nodes" will get the reader to dive into the details. So perhaps saying "Run Aptos nodes" work better?

Reply:
Done!